src/nsntrace.c

```diff
@@ -205,8 +205,12 @@ _nsntrace_start_tracee(struct nsntrace_options *options)
 		uid = getuid();
 		gid = getgid();
 	}
-	setgid(gid);
-	setuid(uid);
+	if (setgid(gid) < 0) {
+		fprintf(stderr, "Unable to set process GID");
+	}
+	if (setuid(uid) < 0) {
+		fprintf(stderr, "Unable to set process UID");
+	}
```

Comment:
Thanks!

So with this code we would report on error that we failed to set the user-id / group id but still carry on.
You do not think it is an fatal operation?